v22.2/drop-external-connection.md

```diff
@@ -0,0 +1,48 @@
+---
+title: DROP EXTERNAL CONNECTION
+summary: The DROP EXTERNAL CONNECTION statement deletes external connections. 
+toc: true
+docs_area: reference.sql
+---
+
+{% include feature-phases/preview.md %}
+
+{% include_cached new-in.html version="v22.2" %} You can use external connections to specify and interact with resources that are external from CockroachDB. When creating an external connection, you define a name for an external connection while passing the provider URI and query parameters. The `DROP EXTERNAL CONNECTION` statement allows you to delete external connections.
+
+You can also use the following SQL statements to work with external connections:
+
+- [`CREATE EXTERNAL CONNECTION`](create-external-connection.html)
+- [`SHOW CREATE EXTERNAL CONNECTION`](show-create-external-connection.html)
+
+## Required privileges
+
+Users must have the [`admin` role](security-reference/authorization.html#admin-role) to drop an external connection.
```

Comment:
It's not necessary for user to be admin to be able to drop an EC. There is a `DROP` privilege similar to the `USAGE` privilege that can be granted to a non-admin user to be able to drop the EC. `admin` can drop any EC withouth the `DROP` privilege.